com.unity.render-pipelines.universal/Documentation~/upgrade-guide-2021-2.md

```diff
@@ -72,11 +72,11 @@ As a result, to match exactly shadow outlines from earlier revisions, the parame
 
 ### Intermediate Texture
 
-Previously, URP would force rendering to go through an intermediate renderer if the Renderer had any Renderer Features active. On some platforms, this has significant performance implications. Due to that, Renderer Features are now expected to declare their inputs using `ScriptableRenderPass.ConfigureInput`. This information is used to decide automatically whether rendering via an intermediate texture is necessary.
+URP would force rendering to go through an intermediate renderer if the Renderer had any Renderer Features active. On some platforms, this has significant performance implications. Due to that, a new property **Intermediate Texture** has been added to the Universal Renderer.
 
-For compatibility reasons, a new property **Intermediate Texture** has been added to the Universal Renderer. This allows for either using the new behaviour, or to force the use of an intermediate texture. The latter should only be used if a Renderer Feature does not declare its inputs using `ScriptableRenderPass.ConfigureInput`.
+When using the new auto intermediate texture mode, render pipeline will decide automatically whether rendering via an intermediate texture is necessary. The decision is based on Renderer Features's input declaration using `ScriptableRenderPass.ConfigureInput`. 
 
-All existing Universal Renderer assets that were using any Renderer Features (excluding those included with URP) are upgraded to force the use of an intermediate texture, such that existing setups will continue to work correctly. Any newly created Universal Renderer assets will default to the new behaviour.
+All newly created Universal Renderer assets will default to the old behaviour(`IntermediateTextureMode.Always`) for compability reason.
```

Comment:
How come the upgrade guide is different from the `master` PR?

Reply:
Reverted the doc change for now in: [bb5251d](https://github.com/Unity-Technologies/Graphics/pull/7280/commits/bb5251d4aa9d7be4b75209505aae42e93c79d96b)

Master branch is currently "frozen" waiting for srp2core. So I am not able to update master->backport at the moment. Reverted the doc change for now and once srp2core work is finished, will do the standard master PR -> backport process for this upgrade guide.